components/butterfly/src/server/mod.rs

```diff
@@ -1575,21 +1578,6 @@ mod tests {
                   .expect("Server failed to start");
         }
 
-        #[test]
-        fn invalid_addresses_fails() {
-            let swim_listen = "";
-            let gossip_listen = "";
-            let member = Member::default();
-            assert!(Server::new(&swim_listen[..],
-                                &gossip_listen[..],
-                                member,
-                                Trace::default(),
-                                None,
-                                None,
-                                None,
-                                Box::new(ZeroSuitability),).is_err())
-        }
```

Comment:
There's no way for there to be invalid addresses any longer since `swim_addr` and `gossip_addr` are proper `SocketAddr` types now.